### PR TITLE
add prow presubmission/postsubmission scripts.

### PR DESCRIPTION
**Pull request by xiaolanz**
address #17

### CODE REVIEW

**Comment by xiaolanz** [comment]
@sebastienvas  How to test these scripts? I just copied from mixer.

**Comment by sebastienvas** [comment]
In order to test on Prow, you need to add a config [here](https://github.com/istio/test-infra/blob/master/prow/config.yaml) for your jobs, ie broker-presubmit and broker-postsubmit.

**Comment by sebastienvas** [comment]
Once you do that they'll show up in the PR and prow.istio.io

File: bin/publish-docker-images.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    h) HUBS="${OPTARG}";;
+    t) TAGS="${OPTARG}";;
+    *) usage;;
+  esac
+done
+
+IFS=',' read -ra TAGS <<< "${TAGS}"
+IFS=',' read -ra HUBS <<< "${HUBS}"
+
+# Building grafana image
```
Comment (sebastienvas) [inline bin/publish-docker-images.sh]: You probablyu don't need that.

**Comment by xiaolanz** [comment]
I should do that after I merge this PR. right?

**Comment by sebastienvas** [comment]
No before. You can add the skip_report: True, if you don't want other PR so get affected.
https://github.com/istio/test-infra/blob/master/prow/config.yaml#L34

**Comment by xiaolanz** [comment]
I've merged PR #425. How can I test now?

**Comment by sebastienvas** [comment]
I fixed codecov and set the token in Prow. Prow finally started
https://k8s-gubernator.appspot.com/build/istio-prow/pull/istio_broker/20/broker-presubmit/2/

**Comment by sebastienvas** [comment]
FYI - postsubmit is passing as well
https://k8s-gubernator.appspot.com/build/istio-prow/broker-postsubmit/1/

Thanks for doing this ! I'll remove the skip_report.

**Pull request merged**